app/src/main/res/layout/activity_crime_pager.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.viewpager.widget.ViewPager xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Hmm, maybe we can use https://developer.android.com/jetpack/androidx/releases/viewpager2